announcement/announcement-api/api/src/bundle/announcement.properties

```diff
@@ -230,6 +230,7 @@ revise.group.list.desc=Description
 ### (moot) revise.group.list.sortdescasc=Sort by group description ascending
 ### (moot) revise.group.list.sortdescdesc=Sort by group description descending
 
+revise.notify_alert = <strong>Email notifications will not be sent</strong> unless this site is published before posting the announcement; however, before clicking Publish Now above, copy your announcement text or post the announcement as hidden (in draft mode) to avoid losing your work.
```

Comment:
Here is how I would phrase this:

<strong>Email notifications will not be sent</strong> while this site is unpublished.

Reply:
That works for me. Later today I'll implement a force push to incorporate that edit. Thanks.